docs/website/root/networks-matrix.md

```diff
@@ -37,6 +38,7 @@ Here is an updated list of all **Mithril networks**, including their configurati
 
 | Information | -
 |------------|------------
+| **Cardano node version** | 8.1.2
```

Comment:
Can you put this information below `Cardano magic Id`?
Also could you make the range mention a bit more clear like:
```suggestion
| **Cardano node version** | 8.1.2+ for restoration
```